.travis.yml

```diff
@@ -26,4 +26,13 @@ before_install:
       -in .travis/pulp-infra.enc -out ~/.ssh/pulp-infra -d
     - sudo chmod 600 ~/.ssh/pulp-infra
 notifications:
-  irc: "chat.freenode.net#pulp-build"
+  irc:
+    channels:
+      - "chat.freenode.net#pulp-build"
+    template:
+      - "%{repository_slug}#%{build_number} (%{branch} - %{commit} : %{author} : %{commit_subject})"
+      - "Change view : %{compare_url}"
+      - "Build details : %{build_url}"
+      - "%{message}"
```

Comment:
Just curious: how is this different than the default and why change it up?

Reply:
Added the commit_subject - makes it easier to know what the PR is at a glance (imnsho, anyway). I expect the template will get tweaked over time as we discover what will make it most useful for us.